corehq/apps/style/static/app_manager/less/new_appmanager/content.less

```diff
@@ -21,6 +21,13 @@
   &.formdesigner-content-wrapper {
     padding: 0;
   }
+  #formdesigner {
+    background-color: @cc-brand-low !important;
+    .fd-content-right .fd-head,
+    .fd-content-right .fd-form-actions {
+      margin-right: -2px;
```

Comment:
There is a 2px white line between the right edge of form player and the right edge of the screen. This gets rid of it. If I make the container 2px bigger, all of formplayer bumps down to the next line